docs/waivers.md

```diff
@@ -26,13 +26,13 @@ Waiver files are [input files](https://www.inspec.io/docs/reference/inputs/) wit
 ```yaml
 control_id:
   expiration_date: YYYY-MM-DD
-  skip: true
+  run: true
   justification: "reason for waiving this control"
 ```
 
 + `expiration_date` is optional. Absence means the waiver is permanent.
-+ `skip` is optional. Absence means the control will run and be
-  reported, but failures in it won't make the overall run fail.
++ `run` is optional. Presence means the control will run and be
+  reported, but failures in it won't make the overall run fail. You may use any of yes, no, true or false.
```

Comment:
"Presence" is not the correct opposite in this case... more like Presence and a false value... don't know how to make that more succinct. Might want to throw @mjingle at that one.

Reply:
I added more clarification - I agree it needs some eyes.